wdtk-rdf/src/main/java/org/wikidata/wdtk/rdf/Vocabulary.java

```diff
@@ -196,6 +196,14 @@
 	}
 
 	/**
+	 * Class for Wikibase rank.
```

Comment:
This is not a class but a property. Properties are defined further down in the file.